firefox-ios/Ecosia/UI/Account/SeedProgressManager/UserDefaultsSeedProgressManager.swift

```diff
@@ -120,42 +121,23 @@ public final class UserDefaultsSeedProgressManager: SeedProgressManagerProtocol
 
         // Determine the effective max seeds and level to enforce (capped or classic)
         let effectiveMaxLevel = maxCappedLevel ?? seedLevels.count
-        let effectiveMaxRequiredSeeds = maxCappedSeeds ?? standardMaxRequiredSeeds
+        var effectiveMaxRequiredSeeds = maxCappedSeeds ?? standardMaxRequiredSeeds
+
+        effectiveMaxRequiredSeeds = maxSeedsForLoggedOutUsers
```

Comment:
This feels strange since we define the variable and always straight after redefine it... Why don't we directly use the final value?

Reply:
Indeed! Just simplified it, thanks!